app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -400,7 +400,7 @@ def process_vm_guest_devices(vm)
       dvds = vm[:Properties][:Props][:VirtualDVDDrives]
       return [] if dvds.empty?
 
-      dvdprops   = dvds[0][:Props]
+      dvdprops   = dvds[0][:Props] || dvds[0][:MS]
       connection = dvdprops[:Connection]
```

Comment:
@bronaghs Probably not. Mostly I was trying to deal with the specs. Just changing :Props to :MS in our yaml file didn't seem to work, so I was trying to handle both possibilities.